rosbridge_server/launch/rosbridge_websocket_client.launch

```diff
@@ -17,7 +17,7 @@
   <group unless="$(arg ssl)">
     <node name="rosbridge_websocket_client" pkg="rosbridge_server" type="rosbridge_websocket_client" output="screen">
       <param name="authenticate" value="$(arg authenticate)" />
-      <param name="server_address" value="$(arg url)"/>
+      <param name="server_uri" value="$(arg uri)"/>
```

Comment:
@Shokman the url parameter was changed to uri here but not in the argument. Also, this is called from [here](https://github.com/creativa77/rocon_web_proxy/blob/master/applications/launch/talker.launch) so it needs to be updated accordingly as well